apps/web/components/connected/fiskistofa/ShipSearchBoxedInput/translation.strings.ts

```diff
@@ -0,0 +1,44 @@
+import { defineMessages } from 'react-intl'
+
+export const translation = defineMessages({
+  shipDetailsHref: {
+    id: 'web.fiskistofaShipSearchBoxedInput:shipDetailsHref',
+    defaultMessage: '/v/gagnasidur-fiskistofu',
+    description: '/v/gagnasidur-fiskistofu',
+  },
+  shipDetailsNumberQueryParam: {
+    id: 'web.fiskistofaShipSearchBoxedInput:shipDetailsNumberQueryParam',
+    defaultMessage: 'nr',
+    description: 'nr',
+  },
+  shipSearchHref: {
+    id: 'web.fiskistofaShipSearchBoxedInput:shipSearchHref',
+    defaultMessage: '/s/fiskistofa/skipaleit',
+    description: '/s/fiskistofa/skipaleit',
+  },
+  label: {
+    id: 'web.fiskistofaShipSearchBoxedInput:label',
+    defaultMessage: 'Skipaskrárnúmer eða nafn skips',
+    description: 'Skipaskrárnúmer eða nafn skips',
+  },
+  placeholder: {
+    id: 'web.fiskistofaShipSearchBoxedInput:placeholder',
+    defaultMessage: '',
```

Comment:
I guess we don't want to display a placeholder, what does it mean when this is an empty string. Will the id be displayed?